src/style.css

```diff
@@ -1,3 +1,64 @@
 body {
   margin: 0;
 }
+
+a,
+h1,
+h2,
+h3,
+p,
+ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
```

Comment:
The `list-style: none;` property is applied to all elements specified (a, h1, h2, h3, p, ul), but it only affects lists (`ul`, `ol`). It's better to apply it only to the necessary elements to avoid confusion.